client/src/app/tabs/Linting.js

```diff
@@ -28,22 +29,24 @@ export function Linting(props) {
 
   const { panel = {} } = layout;
 
-  const errors = linting.filter(({ category }) => category === 'error').length;
+  const errors = linting.filter(({ category }) => category === 'error').length,
+        warnings = linting.filter(({ category }) => category === 'warn').length;
 
   return <Fill slot="status-bar__file" group="9_linting">
     <button
       className={ classnames(
         css.Linting,
         'btn',
         { 'btn--active': panel.open && panel.tab === 'linting',
-          'has-errors': errors > 0
+          'has-errors': errors > 0,
+          'has-warnings': warnings > 0
         }
       ) }
       onClick={ onToggleLinting }
       title="Toggle Errors"
```

Comment:
Tooltip should also say "Toggle problems view"

Reply:
👍🏻